modules/ROOT/pages/aurads/tutorials/arrow-example.adoc

```diff
@@ -85,12 +135,15 @@ We can now time the loading process again. On an 8 GB AuraDS instance, this shou
 gds.graph.ogbn.load("ogbn-arxiv")
 ----
 
+With Arrow disabled by adding `arrow=False` to the `GraphDataScience` constructor, the same loading process would take more than 1 minute.
 Therefore, with this dataset, Arrow provides at least a 2x speedup.
 
 == Cleanup
 
 [source, python]
 ----
-gds.graph.get("ogbn-arxiv").drop()
+article_graph.drop()
+ogbn_arxiv.drop()
```

Comment:
seeing this here. Since 1.7 you can also use `Graph` together with `with` statements.
Then you could get rid of this part and also makes it easier to rerun the cells above

Reply:
That's cool! I can surely do that for the OGBN graph. As for the other one, if I use it I cannot split the loading and streaming sections (not a big deal, but I would have to "collapse" the examples and replace the section break with comments). Would that read better?